spec/dish_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'dish'
+
+describe Dish do
+
+	subject { described_class.new("Pizza",9.99) }
+	
+	describe '#initialize' do
+
+		it 'has a name' do
+			expect(subject.name).to be_a(String)
+		end
+
+		it 'has a price' do
+			expect(subject.price).to be_a(Float)
```

Comment:
As above.  Furthermore, in general you should test behaviour and not types; i.e. test what an object _does_ and not what an object _is_. (duck typing)